go.mod

```diff
@@ -160,6 +161,7 @@ require (
 	google.golang.org/grpc v1.44.0 // indirect
 	google.golang.org/protobuf v1.27.1 // indirect
 	gopkg.in/square/go-jose.v2 v2.5.1 // indirect
+	k8s.io/cri-api v0.24.0-alpha.1 // indirect
```

Comment:
It's being pulled in because we're using 'github.com/containerd/containerd/pkg/cri/' for the annotations I take it... I'd prefer to just redefine the constants if anything like you said. @AkihiroSuda Would you be opposed to adding in hcsshim as a dependency here (not in this PR but in the future)? There's some other annotations that would be useful for hyper-v, but if not we can redefine all the same.

Reply:
Hcsshim dependency is probably fine